bigbluebutton-html5/imports/utils/logoutRouteHandler.js

```diff
@@ -2,13 +2,11 @@ import Auth from '/imports/ui/services/auth';
 
 const logoutRouteHandler = () => {
   Auth.logout()
-    .then((logoutURL = window.location.origin) => {
-      const protocolPattern = /^((http|https):\/\/)/;
-
-      window.location.href =
-        protocolPattern.test(logoutURL) ?
-          logoutURL :
-          `http://${logoutURL}`;
+    .then((logoutURL) => {
+      if (logoutURL) {
+        const protocolPattern = /^((http|https):\/\/)/;
+        window.location.href = protocolPattern.test(logoutURL) ? logoutURL : `http://${logoutURL}`;
```

Comment:
@Tainan404 is there a reason why thisfllback  was `http` and not `https`?

Reply:
I made it to the case of a non url be provided, like a IP (192.168.100.1) or a domain (example.com), so I redirect it as http because the browser turns to https if it's provided.

Reply:
Hmm, interesting, thanks!